Dockerfile

```
@@ -16,6 +16,9 @@ COPY Gemfile Gemfile.lock .ruby-version /app/
 # We reference the engines in the Gemfile, so we need them to be there, too
 COPY vendor/engines /app/vendor/engines
 
+# Install Bundler 2
+RUN gem install bundler --version=2.3.11
```

Instead of doing this, can we read the version of bundler from the `Gemfile.lock`?
```suggestion
RUN grep "BUNDLED WITH" Gemfile.lock; if [ $? -eq 0 ]; then gem install bundler -v "$(grep -A 1 "BUNDLED WITH" Gemfile.lock | tail -n 1)"; fi
```

---

I've updated this and am able to get through most of the Dockerfile steps, though because I'm on M1, I'm unable to fully run docker-compose.

Can someone double check this and see if they have any issues?